common/lib/xmodule/xmodule/course_module.py

```diff
@@ -335,6 +335,15 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_(
+            "Enter the URL for the official course Facebook group. "
+            "If you provide a URL, the edX mobile app includes a button that students can tap to access the group."
+        ),
```

Comment:
Well, I am not an expert on open source vs. not yet ready for open source, but here's my thinking FWIW.
The name of the app is "edX". See https://play.google.com/store/apps/details?id=org.edx.mobile&hl=en
If you provide a URL in this field today, without making any other changes, would it have an effect on any other mobile app besides the edX one? 
Until the mobile app is completely open, I don't know how we can make this platform agnostic.